dir.props

```diff
@@ -306,6 +306,14 @@
     <SymbolPackageOutputPath Condition="'$(SymbolPackageOutputPath)'==''">$(PackageOutputPath)symbols/</SymbolPackageOutputPath>
   </PropertyGroup>
 
+  <!--
+    Remove test-only dependencies during source-build: disable project that brings in XUnit.Runtime
+    dependencies through 'ReferenceFromRuntime' property.
+  -->
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' == 'true' and '$(BuildTests)' != 'true'">
+    <ProjectExclusions Include="$(SourceDir)CoreFx.Private.TestUtilities/**/*.csproj" />
+  </ItemGroup>
```

Comment:
Shouldn't that go into `src\Directory.Build.props`?

cc @weshaggard

Reply:
I suppose `src/dir.props` would also work, if that's preferred. (Nothing in there now, but it does exist and it's in the chain.)

Reply:
sure, I forgot that in 2.1 we don't have the Directory.Build.props changes.